### PR TITLE
ci: make it work under OpenResty 1.19

### PR DESCRIPTION
**Pull request by spacewander**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

**Review by moonming — APPROVED** [review]

File: t/lib/server.lua [hunk 1]
```diff
@@ -357,6 +357,7 @@ function _M.log()
     ngx.req.read_body()
     local body = ngx.req.get_body_data()
     local ct = ngx.var.content_type
+    print(ct)
```
Comment (nic-chen) [inline t/lib/server.lua:360]: is it a debug?

Comment (spacewander) [reply t/lib/server.lua]: @nic-chen 
Fixed.

**Comment by nic-chen** [comment]
It seems that these changes are only suitable for 1.19. 

Do we need to test under 1.18 at the same time?

Thanks!

File: t/node/upstream-node-dns.t [hunk 2]
```diff
@@ -253,20 +253,27 @@ location /t {
 GET /t
 --- grep_error_log eval
 qr/dns resolver domain: \w+.com to 127.0.0.\d|call \/hello|proxy request to 127.0.0.\d:1980/
---- grep_error_log_out
-call /hello
+--- grep_error_log_out eval
```
[the remark this answers is not in the document]
Comment (spacewander) [reply t/node/upstream-node-dns.t:256]: @membphis 
Yes. The dns resolver order is different in OpenResty 1.19.

**Review by membphis — APPROVED** [review]
LGTM

**Comment by membphis** [comment]
I do not know why the status is `waiting for ... reported`.

image

**Review by nic-chen — APPROVED** [review]

**Comment by spacewander** [comment]
@membphis 
Maybe we should disable the outdated check in the protected branch settings of the repository settings? See:
https://github.community/t/github-actions-pull-requests-always-include-some-outdated-checks/16157/2

**Review by gxthrj — APPROVED** [review]

**Pull request merged**